src/query/api/v1/handler/database/create.go

```diff
@@ -205,7 +205,8 @@ func (h *createHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	nsRegistry, err = h.namespaceAddHandler.Add(namespaceRequest)
+	opts := handler.NewServiceOptions("kv", r.Header, nil)
```

Comment:
Why are you passing `"kv"` as the service name? Shouldn't it be M3DB?

Reply:
Yeah I didn't quite know what to pass here. The namespace handler doesn't really use the `service` value it gets passed (only placement cares about that), as namespaces are stored at `_kv/$env/$key`. Because it's controlling KV flags I passed `kv` as the "service name" but can change this to `m3db` if it reads less awkwardly. Even `""` would probably be fine

Reply:
Yeah I think the reason the namespace handler doesn't use a service value is because the only service that has namespaces is M3DB. Still I think it would read a little more cleanly if we passed the M3DB constant here, and then maybe a guard statement in the namespace handler to make sure the service is set to M3DB or return an error